vignettes/pkgdown.Rmd

```diff
@@ -54,27 +54,6 @@ template:
 
 You can learn more about controlling the appearance of your site in `vignette("customise")`.
 
-### Language
```

Comment:
make sure this vignette refer to the non-english vignette?

Reply:
I think people are more likely to see it in the articles drop down than in the getting started vignette.